handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -381,7 +393,32 @@
             TLSV13_SUPPORTED = false;
             IS_BORINGSSL = false;
             EXTRA_SUPPORTED_TLS_1_3_CIPHERS = EmptyArrays.EMPTY_STRINGS;
+            EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING = StringUtil.EMPTY_STRING;
+        }
+    }
+
+    static String checkTls13Ciphers(InternalLogger logger, String ciphers) {
+        if (IS_BORINGSSL && !ciphers.isEmpty()) {
+            assert EXTRA_SUPPORTED_TLS_1_3_CIPHERS.length > 0;
+            Set<String> boringsslTlsv13Ciphers = new HashSet<String>();
+            Collections.addAll(boringsslTlsv13Ciphers, EXTRA_SUPPORTED_TLS_1_3_CIPHERS);
+            boolean ciphersNotMatch = false;
+            for (String cipher: ciphers.split(",")) {
+                if (boringsslTlsv13Ciphers.isEmpty()) {
+                    ciphersNotMatch = true;
```

Comment:
This logic still does not detect a case, when `ciphers` contain:
```
["something_else", "TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384" , "TLS_CHACHA20_POLY1305_SHA256"]
```
iteration over the list above will make `boringsslTlsv13Ciphers` empty and keep the `ciphersNotMatch` flag `false`, which won't trigger the logger.

Consider also checking the result of `boringsslTlsv13Ciphers.remove(cipher)` to know when nothing was removed from the set.